page/upgrade-guide/1.14.md

```diff
@@ -0,0 +1,75 @@
+<script>{
+	"title": "jQuery UI 1.14 Upgrade Guide",
+	"toc": true
+}</script>
+
+## Overview
+
+This guide will assist in upgrading from jQuery UI 1.13.x to jQuery UI 1.14.x. All changes are listed below, organized by plugin, along with how to upgrade your code to work with jQuery UI 1.14.
+
+## Reminder: API Redesigns
+
+jQuery UI 1.12 introduced API redesigns for Button, Buttonset, Dialog, Draggable, Droppable, Menu, Mouse, Resizable, Selectable, Sortable, Tabs, Tooltip, and Effects. You can read about the API redesign process on the
+[jQuery UI Blog](https://blog.jqueryui.com/2011/03/api-redesigns-the-past-present-and-future/). Although the redesigns introduced breaking changes, **1.12, 1.13 & maintain a lot of compatibility with the 1.11 API by default.** This is accomplished by rebuilding the 1.11 API on top of the 1.12/1.13/1.14 API. However, since 1.14, the default behavior is only loading the new API. If you would like to load the legacy 1.11 API as well, you can set the `$.uiBackCompat` flag to `true`.
+
+```html
+<script src="jquery.js"></script>
+<script>$.uiBackCompat = true;</script>
+<script src="jquery-ui.js"></script>
+```
+
+<p class="note">If you find a regression from the 1.11 API, please report it in the <a href="https://github.com/jquery/jquery-ui/issues/new">bug tracker</a>. Even though the 1.11 API is deprecated, it's important for 1.14 releases not to regress so that users are encouraged to upgrade even if they're not ready to use the new APIs.</p>
+
+## General changes
+
+The main focus of this release was reducing the maintenance. This resulted in a few breaking changes.
+
+### Reduced browser support
+
+Only the latest version of Chrome, Firefox, Safari & Edge are officially supported; there is no support for any version of IE and Edge Legacy. Contrary to what was done in past releases, code supporting unsupported browsers has been deleted.
+
+### Reduced jQuery Core support
+
+Only the latest jQuery version within each major version of jQuery Core is supported. To make updates easier, we also run tests with a few other relatively recent versions, but we don't guarantee continued support for them in all jQuery UI releases in the `1.14.x` line. Each jQuery UI version will document on which Core versions it was tested - both in the release blog post and in the changelog page at https://jqueryui.com/changelog/.
+
+### The backwards compatibility flag is off by default
+
+Backwards compatibility with the 1.11 API is disabled by default. To re-enable it (restoring the default 1.13 behavior), set the `jQuery.uiBackCompat` flag to true.
+
+### Removed Core APIs
```

Comment:
I would maybe omit the word "Core" here and in "Newer Core features" below especially since we also talk about "jQuery Core".

In the "Newer Core features" section the same shortcut word refers to two different things only two lines apart.

Reply:
I think I'll rename jQuery Core to jQuery consistently since Core has a meaning in UI, it's a module.

We don't have lots of APIs removed so I can also drop it from here, though.